hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/QueryConfigModule.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.contract.impl.exec;
+
+import static java.util.Objects.requireNonNull;
+
+import com.hedera.node.app.service.contract.impl.annotations.QueryScope;
+import com.hedera.node.app.spi.workflows.QueryContext;
+import com.hedera.node.config.data.ContractsConfig;
+import com.swirlds.config.api.Configuration;
+import dagger.Module;
+import dagger.Provides;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+@Module
+public interface QueryConfigModule {
+    @Provides
+    @QueryScope
+    static Configuration provideConfiguration(@NonNull final QueryContext context) {
+        return requireNonNull(context).configuration();
+    }
+
+    @Provides
+    @QueryScope
+    static ContractsConfig provideContractsConfig(@NonNull final Configuration configuration) {
+        return requireNonNull(configuration).getConfigData(ContractsConfig.class);
+    }
```

Comment:
There's no need to use dagger for these configurations. The pattern has been to pull configurations from the appropriate context instead at the time they're needed (i.e. inside the method with the QueryContext)

Reply:
I was following the pattern that Michael established for the transaction flow but seems excessive.  I'll remove.

Reply:
Dagger isn't needed for the `Configuration` object itself either